plugins/services/src/js/components/forms/MultiContainerNetworkingFormSection.js

```diff
@@ -516,10 +516,15 @@ class MultiContainerNetworkingFormSection extends mixin(StoreMixin) {
         const name = overlay.getName();
 
         return {
+          enabled: overlay.info.enabled,
```

Comment:
🎨 You could omit these additional props if you filter the list of overlays first.